.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Team responsible for Fleet Server
-* @elastic/fleet @elastic/elastic-agent-control-plane
```

Comment:
fyi @kpollich

Reply:
Thanks for the ping, change looks good to me.